dotcom-rendering/src/types/branding.ts

```diff
@@ -51,6 +51,11 @@ export type CollectionBranding =
 			 */
 			isFrontBranding: boolean;
 			branding: Branding;
+			/**
+			 * In certain circumstances a collection might display the branding only on its own container.
+			 * is eligible to display a brand logo.
+			 */
+			isContainerBranding: boolean;
```

Comment:
I'd be inclined to name this [`isConfiguredForBranding`](https://github.com/guardian/frontend/blob/main/common/app/model/facia/PressedCollection.scala#L76) keeping the `frontend` name.